build/util/Dockerfile

```diff
@@ -17,12 +17,18 @@ RUN groupadd -g $ARGOCD_USER_ID argocd && \
     apt-get update && \
     apt-get dist-upgrade -y && \
     apt-get install -y \
-    git git-lfs tini curl python3-pip gpg tzdata && \
+    git git-lfs tini curl python3-pip gpg tzdata unzip && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install the AWS CLI
-RUN pip3 install awscli
+# Create a symlink for python to point to python3
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+# Install AWS CLI v2
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
+    unzip awscliv2.zip && \
+    ./aws/install && \
+    rm -rf awscliv2.zip aws
```

Comment:
Why is awscli required?

Reply:
we've had it from a while back, but I'm not exactly sure why it's required? I had to change it here by installing it directly instead of through pip3 because of a breaking change in pip3. I'll send the link for the convo we had about it to you on slack